invenio_app_ils/locations/loaders/jsonschemas/location.py

```diff
@@ -21,3 +56,44 @@ class LocationSchemaV1(RecordMetadataSchemaJSONV1):
     email = fields.Email()
     phone = fields.Str()
     notes = fields.Str()
+    opening_weekdays = fields.List(fields.Nested(OpeningWeekdaySchema))
+    opening_exceptions = fields.List(fields.Nested(OpeningExceptionSchema))
+
+    @post_load
+    def postload_checks(self, data, **kwargs):
+        """Sort exceptions and validate record."""
+        record = self.context["record"]
+
+        # Bijection between weekdays and indices
+        weekday_names = ["monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"]
```

Comment:
I would have created a (fake) "enum" somewhere, to avoid the iteration each time:
```
WEEKDAYS = {
  "monday": 0,
  ...
}
```

Reply:
:+1: for the enum, I forgot it existed.